website/members/admin.py

```diff
@@ -189,6 +201,14 @@ class UserAdmin(BaseUserAdmin):
         ),
     )
 
+    readonly_fields = ("tpay_balance",)
+
+    def tpay_balance(self, obj):
```

Comment:
If you register a separate admin in the payments package with a per user overview of the payments you can also add custom filters for users that have a balance that is 0. 

This way you would have two different admins for users and the balance and this admin is complete decoupled from payments. For 99% of the time this data is not really necessary